be/src/exec/tablet_sink_index_channel.cpp

```diff
@@ -415,6 +415,20 @@ bool NodeChannel::is_full() {
     return false;
 }
 
+void NodeChannel::_reset_cur_chunk(Chunk* input) {
+    int64_t before_consumed_bytes = CurrentThread::current().get_consumed_bytes();
+    _cur_chunk = input->clone_empty_with_slot();
+    int64_t after_consumed_bytes = CurrentThread::current().get_consumed_bytes();
+    _cur_chunk_mem_usage = after_consumed_bytes - before_consumed_bytes;
```

Comment:
`_cur_chunk_mem_usage += after_consumed_bytes - before_consumed_bytes;` ?

Reply:
> `_cur_chunk_mem_usage += after_consumed_bytes - before_consumed_bytes;` ?

this is the reset phase, we can assign values ​​directly without accumulating